lib/msal-browser/test/interaction_client/RedirectClient.spec.ts

```diff
@@ -1954,7 +2044,8 @@ describe("RedirectClient", () => {
             window.sessionStorage.setItem(AccountEntity.generateAccountCacheKey(testAccountInfo), JSON.stringify(testAccount));
 
             await redirectClient.logout(validatedLogoutRequest).then(() => {
-                expect(window.sessionStorage.length).toBe(0);
+                // Interaction in progress
+                expect(window.sessionStorage.length).toBe(1);
```

Comment:
Thinking about this now, even to have any remaining values in sessionStorage after logout could be unacceptable. Maybe this could just be set in memory storage, so that it blocks interaction that may happen before the redirect happens, but is cleared once the redirect has happened?

Reply:
I don't really consider logout to be "complete" until you return from the redirect (where you should be calling handleRedirectPromise to clear any remaining cache values). I'm personally ok with this